src/backend/main.py

```diff
@@ -71,4 +72,6 @@ def run_server(host: str, port: int, reload: bool, workers: int):
 
 if __name__ == '__main__':
     args = vars(parser.parse_args())
+    logging.warning("Starting main.py")
```

Comment:
Wahrscheinlich vergessen rauszunehmen

Reply:
nein, extra drinnen gelassen (und warnung), damit man in beiden arten von log-dateien genaue sehen kann, aber wann ein neue start des scripts war